docs/rnode-api/rnode-openapi.json

```diff
@@ -38,9 +38,13 @@
           "description": "WebApi.scala",
           "url": "https://github.com/rchain/rchain/blob/dev/node/src/main/scala/coop/rchain/node/api/WebApi.scala"
         },
-        "tags": ["blocks"],
+        "tags": [
```

Comment:
Are these changes correspond to API presented in the code? Are they generated from code or should be written manually?

Reply:
> Are these changes correspond to API presented in the code?

Yes. We created a new api endpoint `/api/transaction/{hash}` at https://github.com/rchain/rchain/pull/3433 which is not covered in current openapi json and that's why we need this pr.

> Are they generated from code or should be written manually?
Currently no. But they should be auto-matically generated when we implement a new api.  I used to have a pr https://github.com/rchain/rchain/pull/3128 which is about generating these open API json. But the auto-generated codes are not accurate so @dckc manually code all these things and made a [pr](https://github.com/rchain/rchain/pull/3140) for that.

**Our goal in the future is to fix this and make these auto-generated with handwriting.**

Reply:
Please open an issue to track the automation goal.

Reply:
> We created a new api endpoint /api/transaction/{hash} at #3433 which is not covered in current openapi json

Please make that more clear in the commit message; for example:

```
fix: /api/transaction/{hash} was missing from open api definition
```

Reply:
> Please open an issue to track the automation goal.

https://github.com/rchain/rchain/issues/2974